spec/integ/bridge.spec.js

```diff
@@ -500,6 +536,36 @@ describe("Bridge", function() {
             });
         });
 
+        it("should keep culled Intents up-to-date with incoming events", function(done) {
```

Comment:
We'll have to go through this one together, I'm a bit confused as to why we shouldn't expect a join and what that has to do with keeping intents up-to-date